packages/eslint-config/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@penskemediacorp/eslint-config",
+  "version": "1.0.0-alpha.0",
+  "description": "Eslint base config and customizations.",
+  "keywords": [
+    "eslint",
+    "js"
+  ],
+  "author": "escobar022 <escobar022@gmail.com>",
+  "homepage": "https://github.com/penske-media-corp/pmc-larva#readme",
+  "license": "ISC",
+  "main": "index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/penske-media-corp/pmc-larva.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "bugs": {
+    "url": "https://github.com/penske-media-corp/pmc-larva/issues"
+  },
+  "peerDependencies": {
+    "eslint": "^6.2.2",
+    "eslint-config-wordpress": "^2.0.0",
+    "eslint-loader": "^3.0.0"
```

Comment:
I feel like `eslint-loader` this should be part of the project, not this package – there could be an instance where the config is used for something that doesn't require webpack e.g. Gulp or no transpiling (definitely relevant for Node). What say you?

Reply:
I tried again from scratch, removing eslint-loader works and is cleaner.
peerDependencies looks like it does not work when doing it from scratch so I must have been testing on a cached version so I changed it back to dependencies